data_steward/cdr_cleaner/cleaning_rules/drop_participants_without_completed_basics.py

```diff
@@ -54,26 +57,23 @@
     ON (concept_id = ancestor_concept_id)
   JOIN `{{project}}.{{dataset}}.observation`
     ON (descendant_concept_id = observation_concept_id)
-  JOIN `{{project}}.{{dataset}}._mapping_observation`
-    USING (observation_id)
   WHERE concept_class_id = 'Module'
     AND concept_name IN ('The Basics')
-    AND src_hpo_id = 'rdr'
     AND questionnaire_response_id IS NOT NULL)
 """)
 
 
-class DropParticipantsWithoutPPI(DropMissingParticipants):
+class DropParticipantsWithoutCompletedBasics(DropMissingParticipants):
     """
-    Drops participants without basics survey or EHR data
+    Drops participants who have not completed any of the "The Basics" survey
     """
 
     def __init__(self,
                  project_id,
                  dataset_id,
                  sandbox_dataset_id,
                  table_namer='stage_less'):
-        desc = (f'Sandbox and remove PIDs with no PPI basics or EHR data.'
+        desc = (f'Sandbox and remove PIDs with no PPI basics.'
                 f'Use drop missing participants CR to remove their records.')
 
         super().__init__(issue_numbers=ISSUE_NUMBERS,
```

Comment:
The constructor here also needs to be updated.  Because we moved the rule out of the `COMBINED_CLEANING_CLASSES` list and into the `RDR_CLEANING_CLASSES` list, we should also update the `affected_datasets` variable to reflect this same change.  This is used if someone ever wants to run the reporting tool to gain more info about the cleaning rules.

Also, I think we need to drop the dependency on `RemoveParticipantDataPastDeactivationDate` now.  That rule has been updated to use `Timestamp` objects instead of `Date` objects now, so the previous dependency is no longer an issue.  I'd be happy to describe the prior issue to you in a call if you'd like.  This will also cause an update to your comment in `clean_cdr.py`.